### PR TITLE
Refactora Msgs de Feedback de Saque da Sidebar e Extrato

### PR DESCRIPTION
**Pull request by miguelssrs**
## Contexto
Esse PR troca as mensagens de feedback de Saque da Sidebar e do Extrato. Na sidebar, o texto é mudado e a parte de antecipação é excluida. 

Fora isso, há um fix nos tooltips da Sidebar (que não estavam aparecendo). Também da página de Antecipação e de Extrato para `width: 200px`, já que no PR relacionado à esse do former-kit-skin-pagarme eu removo o `min-width: 200px` para não afetar os tooltips da Sidebar.

Depende de: https://github.com/pagarme/former-kit-skin-pagarme/pull/177 e https://github.com/pagarme/former-kit/pull/266

## Checklist
- [ ] Mudar mensagem de feedback de Saque na Sidebar
- [ ] Mudar mensagem de feedback de Saque na tela de Antecipação
- [ ] Mudar mensagem de feedback de Saque na tela de Extrato
- [ ] Fix nos tooltips para voltarem a aparecer

## Issues linkadas
- [ ] Resolves https://github.com/pagarme/pilot/issues/1223

### Layout:
`layout base`:
image

## Como testar?
- Vá para a branch `refactor/sidebar-feedback-msgs`
- Gere links do former-kit e former-kit-skin-pagarme indo em cada repositório localmente na branch `refactor/tooltip-classnames` para o former-kit e `refactor/sidebar-feedback-msgs` para former-kit-skin-pagarme e rodando `yarn link` 
- Linke ambos os repositórios na Pilot com `yarn link former-kit` e  `yarn link former-kit-skin-pagarme`
- Rode a Pilot! Veja se os tooltips foram consertados e se as mensagens de feedback estão de acordo com o layout. É isso aí! :tada:

### CODE REVIEW

File: packages/pilot/src/components/Operations/style.css [hunk 1]
```diff
@@ -52,6 +52,7 @@
   align-items: center;
   display: flex;
   justify-content: center;
+  width: 200px;
```
Comment (rsmelo) [inline packages/pilot/src/components/Operations/style.css]: precisa desses 200px? removi e não ci muita diferença

Comment (miguelssrs) [reply packages/pilot/src/components/Operations/style.css]: deletei aqui, valeus

**Comment by rsmelo** [comment]
@miguelssrs pode atualizar o former-kit e former-kit-skin-pagarme nesse pr, foi lançado as versões já

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 89976c21017b0883224a51c9cd5bd9836f8ba9cf
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 89976c21017b0883224a51c9cd5bd9836f8ba9cf
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 89976c21017b0883224a51c9cd5bd9836f8ba9cf
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 89976c21017b0883224a51c9cd5bd9836f8ba9cf
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d225d8b45f2b9c44f5d30c575648d68024e6fcc9
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Pull request merged**